tools/cockpit.spec

```diff
@@ -206,7 +206,8 @@ echo '%dir %{_datadir}/cockpit/playground' > tests.list
 find %{buildroot}%{_datadir}/cockpit/playground -type f >> tests.list
 
 %ifarch x86_64 %{arm} aarch64 ppc64le i686 s390x
-%if 0%{?fedora}
+# HACK: Docker not available https://github.com/cockpit-project/cockpit/issues/12670
+%if 0%{?fedora} && 0%{?fedora} <= 30
```

Comment:
This comes back to the question whether docker is broken temporarily or on its way out of Fedora 31. In the former case we should still build/release the package, as it's more confusing/procedural overhead to remove and re-introduce it after the F31 release.

Reply:
Well it's still unclear, and if it is, it won't be for a while.